packages/vue/src/components/user-panel.vue

```diff
@@ -41,7 +41,14 @@ defineProps({
   user: Object,
 });
 </script>
-<style></style>
+<style lang="scss">
```

Comment:
The same question. Why do you use this style block?

Reply:
yep.. moved it